demo/cmd/bookwatcher/bookwatcher.go

```diff
@@ -0,0 +1,204 @@
+package main
+
+import (
+	"encoding/json"
+	"fmt"
+	"io/ioutil"
+	"log"
+	"net/http"
+	"sync"
+	"time"
+
+	"github.com/openservicemesh/osm/demo/cmd/common"
+)
+
+const (
+	bookBuyerPort   = 8080
+	bookStoreV1Port = 8081
+	bookStoreV2Port = 8082
+	bookThiefPort   = 8083
+)
+
+func clearScreen() {
+	fmt.Print("\033[H\033[2J")
+}
+
+func printGreenln(msg string) {
+	fmt.Printf("\033[32m%s\033[0m\n", msg)
+}
+
+func printYellowln(msg string) {
+	fmt.Printf("\033[33m%s\033[0m\n", msg)
+}
+
+func printRedln(msg string) {
+	fmt.Printf("\033[31m%s\033[0m\n", msg)
+}
+
+func getBookBuyer(bookBuyerPurchases *common.BookBuyerPurchases, wg *sync.WaitGroup, errc chan<- error) {
```

Comment:
nit: better find a way to de-dup the code for these 4 get methods.

Reply:
Great point, thanks! Done!